src/Models/Context/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -59,8 +67,11 @@ export const DeeplinkContextReducer: (
                 draft.sceneId = action.payload.sceneId || '';
                 break;
             }
-            case DeeplinkContextActionType.SET_STORAGE_URL: {
-                draft.storageUrl = action.payload.url || '';
+            case DeeplinkContextActionType.SET_STORAGE_CONTAINER_URL: {
+                draft.storageContainerUrl = action.payload.url || '';
+                if (action.payload.url) {
```

Comment:
same here, why do we need the check?

Reply:
Fixed: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/5a2a6ad1e8181718ff0f6fa4143c38f6ad4e383d